python/tskit/trees.py

```diff
@@ -3334,6 +3334,33 @@ def dump_text(
                 )
                 print(row, file=provenances)
 
+    def __repr__(self):
+        ts_rows = [
+            ["Trees", str(self.num_trees)],
+            ["Seq Length", str(self.sequence_length)],
+            ["Sample Nodes", str(self.num_samples)],
+            ["Total Size TODO", util.naturalsize(99999)],
+        ]
+        header = ["Table", "Rows", "Size", "Has Metadata"]
+        table_rows = []
+        for name, table in self.tables.name_map.items():
+            table_rows.append(
+                [
+                    str(s)
+                    for s in [
+                        name.capitalize(),
+                        table.num_rows,
+                        "TODO",
+                        "Yes"
+                        if hasattr(table, "metadata") and len(table.metadata) > 0
+                        else "No",
+                    ]
+                ]
+            )
+        return util.unicode_table(
+            "TSKIT TreeSequence", None, ts_rows
```

Comment:
I'm not sure about the "TSKIT" all-caps here, feels a bit Trumpian. I think we can just say "TreeSequence", right?

Reply:
Sorry been reading too much twitter. Fixed.